packages/js/product-editor/src/components/attribute-control/new-attribute-modal.tsx

```diff
@@ -99,22 +100,52 @@ export const NewAttributeModal: React.FC< NewAttributeModalProps > = ( {
 		scrollAttributeIntoView( values.attributes.length );
 	};
 
+	const hasTermsOrOptions = ( attribute: EnhancedProductAttribute ) => {
+		return (
+			( attribute.terms || [] ).length > 0 ||
+			( attribute.options || [] ).length > 0
+		);
```

Comment:
I prefer like this:
```suggestion
	const hasTermsOrOptions = ( attribute: EnhancedProductAttribute ) => {
		return (
			( attribute.terms && attribute.terms.length > 0 ) ||
			( attribute.options && attribute.options.length > 0 )
		);
	};
```

But it's a nitpick. That's totally fine as well.

Reply:
I totally agree -- your suggestion is better!